account/create_server_rpc.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package account;
+
+option go_package = "./;grpc_account";
+
+message CreateServerRequest {
+	string environment = 1;
+	bool maintenance_mode = 2;
+	string client_id = 3; // * Used in service token requests
+	string game_server_id = 4; // * Used in NEX token requests
+	repeated string title_ids = 5; // * Used in NASC
+	// TODO - "service" is a bit confusing? Should we rename this?
+	string type = 6; // * Either "nex" for a game server or "service" for a 3rd party service
+	string name = 7;
+	string description = 8; // * Futureproofing. Will be used for 3rd parties using our infrastructure
```

Comment:
What would be the purpose of this?

Reply:
Eventually when we open our infrastructure to 3rd party developers, they'll be able to register their servers with us. Ideally then for users to find servers they want to use, we would have some sort of list or hub or something where users can see what servers are available. These descriptions would be used there

Though saying it now, in cases like that we would probably also want some other stuff for filtering like the game name (which may differ from the server name), tags, etc. So maybe we should separate the concept of a 3rd party server and one of our official servers?